pkgs/development/tools/udis86/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, autoreconfHook, python }:
+
+stdenv.mkDerivation rec {
+  pname = "udis86";
+  version = "1.7.2";
+
+  src = fetchurl {
```

to enable automatic updates:
```suggestion
  src = fetchFromGitHub {
    owner = "vmt";
    repo = "udis86";
    rev = "v${version}";
```